config/locales/en.yml

```diff
@@ -104,7 +104,7 @@ en:
   maths_and_physics:
     policy_name: "Claim a payment for teaching maths or physics"
     policy_short_name: "Maths and Physics"
-    claim_description: "claim for a payment for teaching maths or physics"
+    claim_description: "for a payment for teaching maths or physics"
```

Comment:
This string is also used by `app/views/payment_mailer/confirmation_for_multiple_claims.text.erb`, so that will need updating.